pom.xml

```diff
@@ -484,6 +484,7 @@ flexible messaging model and an intuitive client API.</description>
             <exclude>pulsar-client-cpp/lib/checksum/int_types.h</exclude>
             <exclude>pulsar-client-cpp/lib/checksum/crc32c*</exclude>
             <exclude>pulsar-client-cpp/lib/lz4/lz4.*</exclude>
+            <exclude>pulsar-client-cpp/.idea/*</exclude>
```

Comment:
Git ignore will prevent the idea folder to be checked in and this change will prevent the local build to fail